dev-lang/yab/libyab1-1.7.8.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Extended version of the yabasic programming language"
+DESCRIPTION="yab is an extended version of yabasic, a BASIC programming \
+language, with special commands designed for Haiku."
+HOMEPAGE="http://yab.orgfree.com/"
+COPYRIGHT="1995-2006 Marc-Oliver Ihm (yabasic)
+	2006-2009 Jan Bungeroth (yab improvements)
+	2013-2018 Jim Saxton (yab improvements)"
+LICENSE="Artistic"
+REVISION="1"
+SOURCE_URI="https://github.com/bbjimmy/YAB/archive/libyab1.tar.gz"
+CHECKSUM_SHA256="9b76e11e0bd3f2f318f645126fd4f6171d78dc37b69ae0e2035731edcf53055e"
+SOURCE_FILENAME="libyab1.tar.gz"
+SOURCE_DIR="YAB-libyab1"
+
+SOURCE_URI_2="http://yab.orgfree.com/libyab/libyab.so.zip"
+CHECKSUM_SHA256_2="ec4ef57214e7fe010c3646543dc1fdbc02e9303db30e8113e518a299483a790f"
+
+
+ARCHITECTURES="x86_gcc2 !x86 x86_64"
+
+PROVIDES="
+	devel:libyab = $portVersion
+	lib:libyab = $portVersion
+	devel:libyab1 = $portVersion
+	lib:libyab1 = $portVersion
```

Comment:
Should need "$portName = $portVersion" as a first line (haikuporter takes $portName as a string to start the build)
Why not split the lib:* and devel:* into seperate packages as most library packages do?

Reply:
hello, i miss the reference to the changes made by me, furthermore libyab1 does not correspond to the usual logic of the libs. I wrote a recipe that keeps the old libyab.so and creates the new libyab-1.7.8. Furthermore, I have already written bbjimmy that I will adjust the package. the current package only contains yab and the respective libyab. This has the advantage that the packages become cleaner and can be processed separately.